NzSelectModule Can't imports / exports

### Reproduction link
[https://stackblitz.com/edit/nfzztmkg?file=src%2Fapp%2Fapp.component.ts](https://stackblitz.com/edit/nfzztmkg?file=src%2Fapp%2Fapp.component.ts)

### Steps to reproduce
1. go to https://ng.ant.design/components/select/en
2. Session Examples chose any example
3. Click Edit on StackBlitz

In my real project i receive error like this
✘ [ERROR] TS-996002: This import contains errors, which may affect components that depend on this NgModule. [plugin angular-compiler]

    src/app/shared/ant-design/ant-design.module.ts:53:4:
      53 │     NzSelectModule,
         ╵     ~~~~~~~~~~~~~~


✘ [ERROR] TS-996003: This export contains errors, which may affect components that depend on this NgModule. [plugin angular-compiler]

    src/app/shared/ant-design/ant-design.module.ts:86:4:
      86 │     NzSelectModule,
         ╵     ~~~~~~~~~~~~~~

### What is expected?
Can import / export NzSelectModule on version 19.2.0 or next version.

### What is actually happening?
Can't import / export NzSelectModule on version 19.2.0 (I think from version 19.0.0)

| Environment | Info |
|---|---|
| ng-zorro-antd | 19.2.0 |
| Browser | Chrome |

<!-- generated by ng-zorro-issue-helper. DO NOT REMOVE -->

Having the same issue when building the project with the new 19.2.0 version:

```
[ERROR] TS2339: Property 'NzSelectArrowComponent' does not exist on type 'typeof import("/node_modules/ng-zorro-antd/select/select-arrow.component")'.

node_modules/ng-zorro-antd/select/select.module.d.ts:16:381:
      16 │ ...arComponent, typeof i10.NzSelectArrowComponent, typeof i11.NzSe...
```

--- a/components/select/select-arrow.component.ts
+++ b/components/select/select-arrow.component.ts
@@ -10,9 +10,6 @@ import { NzSafeAny } from 'ng-zorro-antd/core/types';
 import { numberAttributeWithInfinityFallback } from 'ng-zorro-antd/core/util';
 import { NzIconModule } from 'ng-zorro-antd/icon';
 
-/**
- * @internal
- */
 @Component({
   selector: 'nz-select-arrow',
   encapsulation: ViewEncapsulation.None,

